### PR TITLE
.gitignore fixes

### PR DESCRIPTION
**Pull request by vytsci**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,17 +1,22 @@
+# Configuration
+/app/config/parameters.yml
+
+# Libraries
+/bower_components
+/vendor
```
Comment (WouterSioen) [inline .gitignore]: the `/<folder>/*` format has better support in command line tools (like `ack` and `ag` for example)

File: .gitignore [hunk 2]
```diff
@@ -1,17 +1,22 @@
+# Configuration
+/app/config/parameters.yml
+
+# Libraries
+/bower_components
```
Comment (carakas) [inline .gitignore]: we need bower_components for the zip install

Comment (vytsci) [inline .gitignore]: can u explain in more detail? :D

Comment (WouterSioen) [inline .gitignore]: We included both the bower components and the compiled sass files for the Fork backend in the repository for multiple reasons:

* easier for everyone to set up fork without the need of compilation/installation tools
* no issues during deployment (needing to bower install and compile sass)
* easier to create the zip download on the fork cms website

**Pull request merged**